src/app/_components/form/storage-location-selector/index.tsx

```
@@ -0,0 +1,71 @@
+"use client";
+
+import { getSelectProps } from "@conform-to/react";
+import { useTranslations } from "next-intl";
+import { useEffect, useRef, type ComponentProps } from "react";
```

ESLint

[error] 'useEffect' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'useEffect' is defined but never used.

(unused-imports/no-unused-imports)